folksonomy/models.py

```diff
@@ -24,8 +24,10 @@ class ProductTag(BaseModel):
 
     @validator('product')
     def product_check(cls, v):
+        if v == '':
```

Comment:
more pythonic (and also catches None case)

```suggestion
        if not v:
```

Reply:
Oh okay. I'll keep this in mind. Thanks!